templates/partials/receipts-tab.html

```diff
@@ -5,9 +5,17 @@
 <section class="page-section" id="section-2" role="tabpanel" aria-hidden="true" aria-labelledby="section-2-heading">
   <div class="container">
     <div class="page-section__intro">
-      <h2 class="section-heading" id="section-2-heading">Analyze Receipts {% include 'partials/cycle-select.html' %}  </h2>
-      <div class="toggles section__intro__left">
-        <span class="label">Compare by:</span>
+      <div class="section__intro__left">
```

Comment:
Hmmm. In this case I meant it as the left-side element on the section intro. I don't mind either way.